### DISCUSSION TITLE
Vary fan speed to distribute wear over time

### DISCUSSION THREAD

**Discussion by duckduckdog** [Rem0o/FanControl.Releases#1258] [Ideas]
This is probably a rare case, but I was blessed to keep an old GTX 660 running for around 8,000 hours or 380 million revolutions. The card bios defaulted to 30% fan speed for 99% of it's life & developed a rattle which would go away if I increased or decreased the fan by 5%.

Adjusting the RPM +/- 100 rpm across the day would have very likely prevented any rattle by distributing the wear more evenly. I'd be surprised if this was a common issue, but it's probably avoidable.

Thank you for releasing the best Fan Controller by far. It's hard to balance control & ease of use, but it was a breeze to figure out.

**Comment by Rem0o** [comment]
I don't really see how varying the RPM would prevent wear. I believe wear is a function of number of revolutions, and or the voltage applied over time.  

You won't "wear a specific RPM". Except if you can somehow show me so factual/scientific evidence where that's the case? Maybe if the fan is unbalanced in the first place and the "resonant frequency" slowly wear downs something... but we're stretching here. 

In any case, Fan Control is meant to control your fans by temperature, where they will ramp up/down, even completely stop depending on the temperature. Wouldn't that, with natural usage, cover your concern?
